Excepcion.cpp

```diff
@@ -0,0 +1,15 @@
+#include "Excepcion.h"
+
+Excepcion::Excepcion(int motivo)
+{
+    this->motivo = motivo;
+}
+
+const char* ExcepcionLectura::what() const throw()
+{
+	switch(motivo)
+	{
+    case 0 : return "Error en la lectura y cargado del ABB."
+    case 1 : return "Error en la lectura y cargado del grafo."
+	};
```

Comment:
Faltan `;`, `break;` y corregir la indentación
Además cuando lo compilé me tiro un warning acá, porque si no es el caso 0 o el 1 no estaríamos devolviendo nada. Se puede agregar un `default` para agarrar el resto de los casos: 
```suggestion
    switch(motivo) {
        case 0 : 
            return "Error en la lectura y cargado del ABB.";
            break;
        case 1 : 
            return "Error en la lectura y cargado del grafo.";
            break;
        default:
            return "Error desconocido.";
            break;
    }
```
O sino declarar una variable antes del switch, modificarla en el caso 0 o 1, y devolverla al final:
```suggestion
    string error = "Error desconocido.";
    switch(motivo) {
        case 0 : 
            error = "Error en la lectura y cargado del ABB.";
            break;
        case 1 : 
            error = "Error en la lectura y cargado del grafo.";
            break;
    }
    return error;
```

Reply:
Corregido